<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageParcelRecordsDialog</class>
 <widget class="QDialog" name="ManageParcelRecordsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Valuation Records for Parcel:</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>88</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>118</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>148</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>345</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>28</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>345</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>23</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>69</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Records</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parcel_type_cbox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>226</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>375</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage Parcel Records</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="record_twidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>89</y>
     <width>330</width>
     <height>241</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>320</number>
   </attribute>
   <column>
    <property name="text">
     <string>Records Information</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>helpButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
